test/appium/tests/marks.py

```diff
@@ -21,3 +21,5 @@
 sign_in = pytest.mark.sign_in
 skip = pytest.mark.skip
 logcat = pytest.mark.logcat
+
+device_nightly = pytest.mark.device_nightly
```

Comment:
btw, the previous mark name I liked more than the current one. 
i.e. battery or battery_consumption

Reply:
I will change it to battery_consumption